aten/src/ATen/native/DistributionTemplates.h

```diff
@@ -64,6 +64,40 @@ at::Tensor& random_impl(at::Tensor& self, at::Generator* generator) {
   return self;
 }
 
+#define show_out_of_bounds_warning(name) \
+  TORCH_WARN(name, " is out of bounds for ", dtype, ". This warning will become an error in version 1.6 release, please fix the code in advance")
+
+static void check_from_to(int64_t from, int64_t to_inc, caffe2::TypeMeta dtype) {
```

Comment:
nit: maybe rename it as check_from_to_in_range

Reply:
renamed